lib/matplotlib/markers.py

```diff
@@ -93,6 +93,8 @@
 (TICKLEFT, TICKRIGHT, TICKUP, TICKDOWN,
  CARETLEFT, CARETRIGHT, CARETUP, CARETDOWN) = list(xrange(8))
 
+_empty_path = Path(np.empty((0, 2)))
```

Comment:
FWIW, there is a fast instantiation route for Paths: https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/path.py#L173.